src/constants.js

```diff
@@ -0,0 +1,18 @@
+export const MediaSizes = {
+    sm: {
+        width: '600px',
+        value: 600,
+    },
+    md: {
+        width: '960px',
+        value: 960,
+    },
+    lg: {
+        width: '1280px',
+        value: 1280,
+    },
+    xl: {
```

Comment:
In order to improve readability, I think we should use SMALL, MEDIUM, LARGE and X_LARGE
also if the value is not used we could put directly:
```
export const MEDIA_SIZES = {
    SMALL: '600px',
    ...
}
```
capital case is for convention
constants are written in this way